integrations/user-auth/shared-session.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Shared Session Auth'
+description: 'Seamlessly share user sessions between your dashboard and your docs'
+---
+
+This method utilizes the session authentication info already stored in your user’s browser to create a seamless documentation experience.
+
+## Implementation
+
+<Steps>
+  <Step title="Create your Info API">
+    Create an API endpoint that uses session authentication to identify users, and responds with a JSON payload following the [UserInfo](./sending-data) format.
+
+    If the API domain does not *exactly match* the docs domain:
+      - Add the docs domain to your API's `Access-Control-Allow-Origin` header (must not be `*`)
+      - Ensure your API’s `Access-Control-Allow-Credentials` header is `true`
+      <Warning>
+        These CORS options only need to be enabled on the *single endpoint* responsible for returning user information. We do not recommend enabling these options on all dashboard endpoints.
+      </Warning>
+  </Step>
+  <Step title="Configure your User Auth settings">
+    Go to your [Mintlify dashboard settings](https://dashboard.mintlify.com/settings/integrations) and add the API URL and your Login URL to your User Auth settings.
+  </Step>
+</Steps>
+
+## Examples
```

Comment:
I think the examples should also go in the "choosing" page.

Reply:
I feel like it makes sense to have the examples here for when people actually need to implement stuff, no?

Reply:
makes sense